pkg/assets/assets.go

```diff
@@ -0,0 +1,67 @@
+package assets
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/coreos/bootkube/pkg/assets/internal"
+)
+
+const (
+	AssetPathCAKey           = "tls/ca.key"
+	AssetPathCACert          = "tls/ca.crt"
+	AssetPathAPIServerKey    = "tls/apiserver.key"
+	AssetPathAPIServerCert   = "tls/apiserver.crt"
+	AssetPathCASecret        = "manifests/ca-secret.yaml"
+	AssetPathAPIServerSecret = "manifests/apiserver-secret.yaml"
+	AssetPathAPIServer       = "manifests/apiserver.yaml"
+)
+
+//go:generate go run templates_gen.go
+//go:generate gofmt -w internal/templates.go
+
+type Asset struct {
+	Name string
+	Data []byte
+}
+
+type Assets []Asset
+
+func (a Assets) Append(asset ...Asset) Assets {
```

Comment:
This method isn't strictly necessary. You can use the builtin `append` function on the `Assets` type.